### ISSUE TITLE
InstabugDialogActivity keeps showing up in the log on Pixel 3a

### ISSUE THREAD

**Issue by KarimFikani** [Instabug/Instabug-Android#246]
### Steps to Reproduce the Problem
Customers with Pixel 3a are reporting that the bug reporting popup keep showing up and would make the app unusable.
I looked at the the logs for that bug report and saw in the logs InstabugDialogActivity "Common.ApplicationLifecycleListener(22905): onActivityStarted:class com.instabug.library.invocation.invocationdialog.InstabugDialogActivity" keeps showing up.

I'm unable to reproduce the issue on an emulator since I don't have access to a Pixel 3a so the logs are available on the instabug dashboard.

### Expected Behavior
Not to popup all the time. Only when a related event is triggered

### Actual Behavior
Popup dialog keeps popping up and making the app unusable

### Instabug integration code
```
 new Instabug.Builder(this, Constants.INSTANCE.getINSTA_BUG_API_KEY())
                .setInvocationEvents(InstabugInvocationEvent.NONE)
                .setCrashReportingState(Feature.State.DISABLED)
                .build();
        BugReporting.setPromptOptionsEnabled(PromptOption.BUG, PromptOption.FEEDBACK);
        BugReporting.setInvocationEvents(InstabugInvocationEvent.SHAKE);

        Instabug.setColorTheme(InstabugColorTheme.InstabugColorThemeLight);
        Instabug.setPrimaryColor(ContextCompat.getColor(this, R.color.cerulean));

        InstabugCustomTextPlaceHolder placeHolder = new InstabugCustomTextPlaceHolder();
        placeHolder.set(InstabugCustomTextPlaceHolder.Key.REPORT_FEEDBACK, getString(R.string.report_feedback));
        placeHolder.set(InstabugCustomTextPlaceHolder.Key.REPORT_BUG, getString(R.string.report_bug));
        Instabug.setCustomTextPlaceHolders(placeHolder);
```

### SDK Version
8.0.8

### Android Version
API 29

### Device Model
Pixel 3a

**Comment by mzelzoghbi** [comment]
Hi @KarimFikani , Can you please try to update to 8.7.0 and tell us if the issue still occurs or not?

**Comment by KarimFikani** [comment]
ok will do. Won't know until after I release and no one reports about the issue. thx!

**Comment by mzelzoghbi** [comment]
@KarimFikani we released 8.7.2 couple of days ago, please give it a try and feel free to reach out to us again if you encountered any issues.

Thank You.
